app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -144,6 +144,16 @@ def api_authorization_token
     User.current.jwt_token!(expiration: 4.hours.to_i, scope: scope)
   end
 
+  def find_object(klass, id = nil, title = nil)
```

Comment:
We can use friendlyId features here: https://github.com/norman/friendly_id?tab=readme-ov-file#usage

Reply:
Can you use `friendly_id` without ActiveRecord? From the description, it looks like it's only for Rail models.

Reply:
Didn't try it, but then the class won't have the `find` and `find_by` methods. 
I don't think it's a problem in this case.